src/sentry/integrations/github/webhook.py

```diff
@@ -515,6 +517,22 @@ def _handle(
                     "merge_commit_sha": merge_commit_sha,
                 },
             )
+
+            if action == "opened" and created:
```

Comment:
the `created` check may be slightly redundant...

but `opened` and `reopened` are different. we could also trigger this on reopen so that the comment gets updated, but we'd probably only want to check the action in that case since the PR likely already exists

<details>
<summary>action types</summary>
<img width="251" alt="Screenshot 2023-11-27 at 14 44 22" src="https://github.com/getsentry/sentry/assets/70817427/a9361a08-904f-4200-8a32-0f1be1eb30a7">
</details>

Reply:
If the PR gets updated and the files we showed are no longer relevant, will the comment update to new errors?

Reply:
ok for now what i said stands. the number of users reopening PRs is low so we'll keep it simple for now